.github/workflows/deploy.yml

```diff
@@ -28,8 +28,16 @@ jobs:
       env:
         CF_API_KEY: ${{ secrets.CF_API_KEY }}
         CF_EMAIL: ${{ secrets.CF_EMAIL }}
-    - name: Upload assets
+    - name: Upload assets (development)
       run: wrangler publish
       env:
         CF_API_KEY: ${{ secrets.CF_API_KEY }}
         CF_EMAIL: ${{ secrets.CF_EMAIL }}
+    - name: Copy wrangler.production.toml to wrangler.toml
+      run: cp wrangler.production.toml wrangler.toml
+    - name: Publish site
+      uses: cloudflare/wrangler-action@1.2.0
+      with:
+        apiToken: ${{ secrets.CF_PRODUCTION_API_TOKEN }}
```

Comment:
@adamschwartz can i get admin access on this repo so i can manage this secret?

Reply:
Of course! You should have admin access now.